As required by the libjpeg license, additions, deletions and changes to
the original files are listed here. Files noted as "Only in jpeg-8c"
were deleted; files noted as "Only in third_party/jpeg/" were added.

To regenerate this file, replace everything after "------" with the output
of this command: diff -w -ru jpeg-8c third_party/jpeg

------

Only in jpeg-8c: Makefile.am
Only in jpeg-8c: Makefile.in
Only in jpeg-8c: aclocal.m4
Only in jpeg-8c: ansi2knr.1
Only in jpeg-8c: ansi2knr.c
Only in jpeg-8c: cderror.h
Only in jpeg-8c: cdjpeg.c
Only in jpeg-8c: cdjpeg.h
Only in jpeg-8c: change.log
Only in jpeg-8c: cjpeg.1
Only in jpeg-8c: cjpeg.c
Only in jpeg-8c: ckconfig.c
Only in jpeg-8c: coderules.txt
Only in jpeg-8c: config.guess
Only in jpeg-8c: config.sub
Only in jpeg-8c: configure
Only in jpeg-8c: configure.ac
Only in jpeg-8c: depcomp
Only in jpeg-8c: djpeg.1
Only in jpeg-8c: djpeg.c
Only in jpeg-8c: example.c
Only in jpeg-8c: filelist.txt
Only in jpeg-8c: install-sh
Only in jpeg-8c: install.txt
Only in jpeg-8c: jconfig.bcc
Only in jpeg-8c: jconfig.cfg
Only in jpeg-8c: jconfig.dj
Only in third_party/jpeg/: jconfig.h
Only in jpeg-8c: jconfig.mac
Only in jpeg-8c: jconfig.manx
Only in jpeg-8c: jconfig.mc6
Only in jpeg-8c: jconfig.sas
Only in jpeg-8c: jconfig.st
Only in jpeg-8c: jconfig.txt
Only in jpeg-8c: jconfig.vc
Only in jpeg-8c: jconfig.vms
Only in jpeg-8c: jconfig.wat
Only in jpeg-8c: jdosabcc.obj
Only in jpeg-8c: jdosamsc.obj
Only in jpeg-8c: jdosaobj.txt
diff -w -ru jpeg-8c/jerror.c third_party/jpeg/jerror.c
--- jpeg-8c/jerror.c	1998-02-21 21:03:16.000000000 +1100
+++ third_party/jpeg/jerror.c	2022-02-11 08:38:08.000000000 +1100
@@ -24,6 +24,8 @@
 #include "jversion.h"
 #include "jerror.h"

+#include <stdlib.h>
+
 #ifdef USE_WINDOWS_MESSAGEBOX
 #include <windows.h>
 #endif
Only in jpeg-8c: jmemansi.c
Only in jpeg-8c: jmemdos.c
Only in jpeg-8c: jmemdosa.asm
Only in jpeg-8c: jmemmac.c
Only in jpeg-8c: jmemname.c
Only in jpeg-8c: jpegtran.1
Only in jpeg-8c: jpegtran.c
Only in jpeg-8c: libjpeg.map
Only in jpeg-8c: libjpeg.txt
Only in jpeg-8c: ltmain.sh
Only in jpeg-8c: makcjpeg.st
Only in jpeg-8c: makdjpeg.st
Only in jpeg-8c: makeadsw.vc6
Only in jpeg-8c: makeasln.v10
Only in jpeg-8c: makecdep.vc6
Only in jpeg-8c: makecdsp.vc6
Only in jpeg-8c: makecfil.v10
Only in jpeg-8c: makecmak.vc6
Only in jpeg-8c: makecvcx.v10
Only in jpeg-8c: makeddep.vc6
Only in jpeg-8c: makeddsp.vc6
Only in jpeg-8c: makedfil.v10
Only in jpeg-8c: makedmak.vc6
Only in jpeg-8c: makedvcx.v10
Only in jpeg-8c: makefile.ansi
Only in jpeg-8c: makefile.bcc
Only in jpeg-8c: makefile.dj
Only in jpeg-8c: makefile.manx
Only in jpeg-8c: makefile.mc6
Only in jpeg-8c: makefile.mms
Only in jpeg-8c: makefile.sas
Only in jpeg-8c: makefile.unix
Only in jpeg-8c: makefile.vc
Only in jpeg-8c: makefile.vms
Only in jpeg-8c: makefile.wat
Only in jpeg-8c: makejdep.vc6
Only in jpeg-8c: makejdsp.vc6
Only in jpeg-8c: makejdsw.vc6
Only in jpeg-8c: makejfil.v10
Only in jpeg-8c: makejmak.vc6
Only in jpeg-8c: makejsln.v10
Only in jpeg-8c: makejvcx.v10
Only in jpeg-8c: makeproj.mac
Only in jpeg-8c: makerdep.vc6
Only in jpeg-8c: makerdsp.vc6
Only in jpeg-8c: makerfil.v10
Only in jpeg-8c: makermak.vc6
Only in jpeg-8c: makervcx.v10
Only in jpeg-8c: maketdep.vc6
Only in jpeg-8c: maketdsp.vc6
Only in jpeg-8c: maketfil.v10
Only in jpeg-8c: maketmak.vc6
Only in jpeg-8c: maketvcx.v10
Only in jpeg-8c: makewdep.vc6
Only in jpeg-8c: makewdsp.vc6
Only in jpeg-8c: makewfil.v10
Only in jpeg-8c: makewmak.vc6
Only in jpeg-8c: makewvcx.v10
Only in jpeg-8c: makljpeg.st
Only in jpeg-8c: maktjpeg.st
Only in jpeg-8c: makvms.opt
Only in jpeg-8c: missing
Only in jpeg-8c: rdbmp.c
Only in jpeg-8c: rdcolmap.c
Only in jpeg-8c: rdgif.c
Only in jpeg-8c: rdjpgcom.1
Only in jpeg-8c: rdjpgcom.c
Only in jpeg-8c: rdppm.c
Only in jpeg-8c: rdrle.c
Only in jpeg-8c: rdswitch.c
Only in jpeg-8c: rdtarga.c
Only in jpeg-8c: readme.dos
Only in jpeg-8c: structure.txt
Only in jpeg-8c: testimg.bmp
Only in jpeg-8c: testimg.jpg
Only in jpeg-8c: testimg.ppm
Only in jpeg-8c: testimgp.jpg
Only in jpeg-8c: testorig.jpg
Only in jpeg-8c: testprog.jpg
Only in jpeg-8c: transupp.c
Only in jpeg-8c: transupp.h
Only in jpeg-8c: usage.txt
Only in jpeg-8c: wizard.txt
Only in jpeg-8c: wrbmp.c
Only in jpeg-8c: wrgif.c
Only in jpeg-8c: wrjpgcom.1
Only in jpeg-8c: wrjpgcom.c
Only in jpeg-8c: wrppm.c
Only in jpeg-8c: wrrle.c
Only in jpeg-8c: wrtarga.c
